cli/js/event_target_test.ts

```diff
@@ -4,8 +4,11 @@ import { test, assertEquals } from "./test_util.ts";
 test(function addEventListenerTest(): void {
   const document = new EventTarget();
 
+  // @ts-ignore
```

Comment:
What are we ignoring here and ignoring.  Feels like something isn't right if this is failing.

Reply:
I have to admit that I was a bit confused by this test.
So with `strictNullChecks` turned on, `null` isn't considered a valid second argument to `document.addEventListener`.
I guess an alternative would be to cast `addEventListener` as `any` like this:
```typescript
  assertEquals((document.addEventListener as any)("x", null, false), undefined);
```
although I see that option as largely equivalent.

I'll gladly change the `// @ts-ignore` to `any` casts if that's what you'd prefer.